package.json

```diff
@@ -1,88 +1,91 @@
-        "Thorsten Scherler <scherler@gmail.com>",
-        "Tom Fennelly <tom.fennelly@gmail.com>",
-        "Cliff Meyers <cliff.meyers@gmail.com>",
-        "Ivan Meredith <ivan@ivan.net.nz>",
-        "Kohsuke Kawaguchi <kk@kohsuke.org>",
-        "Yuta Hiroto <git@about-hiroppy.com>"
-    ],
-    "license": "MIT",
-    "files": [
-        "LICENSE",
-        "README.md",
-        "licenses",
-        "dist"
-    ],
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/jenkinsci/jenkins-design-language.git"
-    },
-    "dependencies": {
-        "moment": "2.13.0",
-        "moment-duration-format": "1.3.0",
-        "react": "15.1.0",
-        "react-addons-css-transition-group": "15.1.0",
-        "react-dom": "15.1.0",
-        "react-material-icons-blue": "1.0.4"
-    },
-    "devDependencies": {
-        "@kadira/storybook": "1.34.0",
-        "babel-eslint": "6.0.5",
-        "babel-preset-es2015": "6.9.0",
-        "babel-preset-react": "6.5.0",
-        "babel-preset-stage-0": "6.5.0",
-        "babelify": "7.3.0",
-        "browserify": "13.0.1",
-        "chai": "3.5.0",
-        "enzyme": "2.4.1",
-        "eslint-plugin-react": "5.2.2",
-        "flow-bin": "0.27.0",
-        "gulp": "3.9.1",
-        "gulp-babel": "6.1.2",
-        "gulp-clean": "0.3.2",
-        "gulp-concat": "2.6.0",
-        "gulp-copy": "0.0.2",
-        "gulp-eslint": "2.0.0",
-        "gulp-less": "3.1.0",
-        "gulp-mocha": "2.2.0",
-        "gulp-rename": "1.2.2",
-        "gulp-sourcemaps": "1.6.0",
-        "gulp-svgmin": "1.2.2",
-        "harp": "^0.20.3",
-        "karma": "0.13.22",
-        "karma-browserify": "5.0.5",
-        "karma-chrome-launcher": "1.0.1",
-        "karma-junit-reporter": "1.1.0",
-        "karma-mocha": "1.0.1",
-        "karma-mocha-reporter": "2.0.4",
-        "karma-phantomjs-launcher": "1.0.0",
-        "mocha": "2.5.3",
-        "normalize.css": "4.1.1",
-        "octicons": "4.2.0",
-        "phantomjs-prebuilt": "2.1.7",
-        "react-addons-test-utils": "15.1.0",
-        "react-router": "2.4.1",
-        "run-sequence": "1.2.1",
-        "watchify": "3.7.0"
-    }
+  "name": "@jenkins-cd/design-language",
+  "jdlName": "jenkins-design-language",
+  "version": "0.0.78-3",
+  "description": "Styles, assets, and React classes for Jenkins Design Language",
+  "main": "dist/js/components/index.js",
+  "scripts": {
+    "storybook": "npm run gulp build && start-storybook -p 9001 --static-dir dist/assets/",
+    "build-storybook": "build-storybook --static-dir dist/assets/ --output-dir website/storybook",
+    "flow": "flow",
+    "gulp": "gulp",
+    "test": "gulp test",
+    "site-build": "harp compile website/ website-build/",
+    "site-server": "harp server website --port 3000",
+    "site-publish": "./site-publish.sh",
+    "preversion": "npm install && gulp",
+    "release": "npm version patch && npm publish && npm run site-publish",
```

Comment:
Yes. npm version will call "preversion" and "gulp" with no args runs a full lint / test / build.